defaults/main.yml

```diff
@@ -36,6 +39,7 @@ airflow_dags_folder: "{{ airflow_home }}/dags"
 airflow_logs_folder: /var/log/airflow
 airflow_child_process_log_folder: "{{ airflow_logs_folder }}/scheduler"
 airflow_environment_file_folder: /etc/sysconfig
+airflow_environment_extra_vars: []
```

Comment:
Could you please add a sample value like in [L208](https://github.com/idealista/airflow-role/blob/master/defaults/main.yml#L208)?

Reply:
done.